src/commands/main.js

```diff
@@ -1,13 +1,10 @@
 const Discord = require('discord.js')
-const dayjs = require('dayjs')
-const relativeTime = require('dayjs/plugin/relativeTime')
-dayjs.extend(relativeTime)
 
-const { system } = require('../utils')
+const { system, relativeTime } = require('../utils')
 const version = require('../../package.json').version
 
 module.exports = async function (message, parent) {
-  let summary = `Dokdo v${version}, discord.js \`${Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${dayjs(system.processReadyAt()).fromNow()}, bot was ready at ${dayjs(parent.client.readyAt).fromNow()}.\n`
+  let summary = `Dokdo v${version}, discord.js \`${Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${relativeTime(system.processReadyAt())}, bot was ready at ${relativeTime(parent.client.readyTimestamp)}.\n`
```

Comment:
relativeTime must pass Date. Not Number(Timestamp)
So replace readyTimestamp to readyAt.

```suggestion
  let summary = `Dokdo v${version}, discord.js \`${Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${relativeTime(system.processReadyAt())}, bot was ready at ${relativeTime(parent.client.readyAt)}.\n`
```